Sources/Rendering/WebGPU/ImageMapper/index.js

```diff
@@ -236,7 +240,7 @@ function vtkWebGPUImageMapper(publicAPI, model) {
 
         const target = iComps ? i : 0;
         const cfun = actor.getProperty().getRGBTransferFunction(target);
-        if (cfun && actor.getProperty().getUseLookupTableScalarRange()) {
+        if (cfun) {
```

Comment:
@sankhesh  Here is where I deleted the test. I think the solution is that this code here and the getTable() call far later down have to use the same range. Right now the getTable call uses the table range while in the old code this part did not. Same idea on the WebGL side.

Reply:
The advantage of using the tfun range is that it tends to not change much. To use the window level to create the texture means rebuilding the texture every time the window or level changes. I'm not sure the details of the change you made so I'll leave it up to you in terms of how you want to change it.